### PR TITLE
Use update 2 images when subscription is enabled.

### PR DESCRIPTION
**Pull request by Aaquiff**
## Purpose
Use update 2 images when subscription is enabled.

### CODE REVIEW

File: advanced/is-pattern-1/templates/_helpers.tpl [hunk 1]
```diff
@@ -57,6 +57,30 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
 
+{{- define "image2" }}
```
Comment (ThilinaManamgoda) [inline advanced/is-pattern-1/templates/_helpers.tpl:60]: what is image2 ?

File: advanced/is-pattern-1/templates/_helpers.tpl [hunk 2]
```diff
@@ -57,6 +57,30 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
...
+image: "docker.wso2.com/{{ .deployment.imageName }}:5.10.0"
+{{- end }}
+
+{{- define "isSubscription" }}
```
Comment (ThilinaManamgoda) [inline advanced/is-pattern-1/templates/_helpers.tpl:64]: Are we using this?

File: advanced/is-pattern-1/templates/_helpers.tpl [hunk 3]
```diff
@@ -57,6 +57,30 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
 
+{{- define "image2" }}
+image: "docker.wso2.com/{{ .deployment.imageName }}:5.10.0"
```
Comment (ThilinaManamgoda) [inline advanced/is-pattern-1/templates/_helpers.tpl:61]: Image version should be 5.11.0 right

**Review by ThilinaManamgoda — CHANGES_REQUESTED** [review]

File: advanced/is-pattern-1/templates/_helpers.tpl (hunk 3 above)
Comment (Aaquiff) [reply advanced/is-pattern-1/templates/_helpers.tpl]: Unused. Removed.

File: advanced/is-pattern-1/templates/_helpers.tpl (hunk 2 above)
Comment (Aaquiff) [reply advanced/is-pattern-1/templates/_helpers.tpl]: Unused. Removed.

File: advanced/is-pattern-1/templates/_helpers.tpl (hunk 1 above)
Comment (Aaquiff) [reply advanced/is-pattern-1/templates/_helpers.tpl]: Unused. Removed.

**Review by ThilinaManamgoda — APPROVED** [review]

**Pull request merged**